docker/Dockerfile

```diff
@@ -1,27 +1,35 @@
-FROM alpine:3.10
+FROM alpine:3.11 AS downloader
```

Comment:
Just a general question: what is the benefit of a multi-stage deployment here if we are using the same image in both places ? As I understand, multi-stage deployments are useful when one would want to build in an image with lots of requirements but finally place the binary in a light image.